src/day6.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <algorithm>
+
+
+std::vector<std::vector<std::string>> readPuzzleInputFromFile(const std::string& fileName) {
```

Comment:
vectors of vectors of ... are always a bit hard to read. You could create a struct to hold the group answers, e.g.:

```cpp
struct GroupAnswers
{
    std::vector<std::string> answers_per_person;
};
```

or you could use a typedef:

```cpp
typedef std::vector<std::string> GroupAnswers;
```